.circleci/config.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.4.3-node-browsers
+    steps:
+      - checkout
+      - run:
+          # This is essential for capybara-webkit
+          name: Install System Dependencies
+          command: sudo apt-get install qt5-default libqt5webkit5-dev gstreamer1.0-plugins-base gstreamer1.0-tools gstreamer1.0-x
+      - run: bundle install --without local
+      - run: rake install
+      - run: rake spec:ci
+      - run: rake cucumber:ci
```

Comment:
might be worth having a `test:ci` task that runs both specs and cucumber features.